tests/Feature/NetworkProviderSyncTest.php

```
@@ -0,0 +1,530 @@
+            'type' => NetworkType::PROVIDER,
+            'server_provider_id' => $this->connection->id,
+            'external_id' => '4711',
+        ]);
+
+        $otherUser = User::factory()->create();
+        $otherUser->ensureHasDefaultProject();
+
+        app(SyncProviderNetworks::class)->forProject($otherUser->currentProject, $other);
+
+        $this->assertDatabaseHas('networks', ['id' => $other->id]);
+    }
```

PHPMD (2.15.0)

[error] The method test_sync_ignores_networks_of_another_project is not named in camelCase. (undefined)

(CamelCaseMethodName)